example/counter/lib/main.dart

```diff
@@ -1,162 +0,0 @@
-// ignore_for_file: avoid_print
```

Comment:
FYI, there might be things for you to crib in this main.dart, but yes, otherwise this is long since dead.